DuckDuckGo/AppDelegate.swift

```diff
@@ -536,7 +536,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
 
         NotificationCenter.default.post(name: AutofillLoginListAuthenticator.Notifications.invalidateContext, object: nil)
-        mainViewController?.clearNavigationStack()
+        // mainViewController?.clearNavigationStack()
```

Comment:
Need to fix this, it was getting in the way of the open URL functionality. I'll update it to avoid this check depending on the URL.